CollapseLauncher/Classes/GameManagement/GameSettings/StarRail/RegistryClass/Model.cs

```diff
@@ -29,6 +29,15 @@ public enum AntialiasingMode // TypeDefIndex: 25409
         TAA = 1,
         FXAA = 2
     }
+
+    public enum CharacterQuality
+    {
+        Placeholder,
+        Low = 1,
+        Placeholder2,
+        Medium = 3,
+        High = 4
+    }
```

Comment:
As per my investigation, the Enum Value for ``CharacterQuality`` should be:
```CSharp
    public enum CharacterQuality
    {
        Low = 2,
        Medium = 3,
        High = 4
    }
```

| Low | Medium | High |
|-----|--------|------|
|  ![image](https://github.com/CollapseLauncher/Collapse/assets/30566970/aa32bdf7-5347-4a5f-b0bd-9cbdba73c966)  |  ![image](https://github.com/CollapseLauncher/Collapse/assets/30566970/d9eb419a-8181-47d9-a3a6-823b79673a1a)   |  ![image](https://github.com/CollapseLauncher/Collapse/assets/30566970/66de70e7-c778-4837-a9d0-24d4a45bddc3)  |

Reply:
Adjusted